.gitignore

```diff
@@ -5,3 +5,4 @@ coverage
 lib/
 *.log
 *~
+.idea/
```

Comment:
this is not required. you can add it to your global gitignore

Reply:
thanks for the feedback, correcting in next commit